app/javascript/packs/user_timelogs/new_timelog.jsx

```diff
@@ -40,8 +40,12 @@ class NewTimelog extends React.Component {
     this.setState({ startTime: event.target.value });
   }
 
-  handleDurationChange(event) {
-    this.setState({ duration: event.target.value });
+  handleHoursChange(event){
+    this.setState({ hours: event.target.value * 60 });
```

Comment:
I would calculate this in duration. Here we have `hours` state (by name) and `minutes` (by value)

Reply:
Don't forget to add new feature for admin/new_timelog as well